README.md

```diff
@@ -3,7 +3,7 @@
 
 [![npm](https://img.shields.io/npm/v/ship-components-grid.svg?maxAge=2592000)](https://www.npmjs.com/package/ship-components-grid)
 [![Build Status](http://img.shields.io/travis/ship-components/ship-components-grid/master.svg?style=flat)](https://travis-ci.org/ship-components/ship-components-grid)
-[![Coverage](http://img.shields.io/coveralls/ship-components/ship-components-grid.svg?style=flat)](https://coveralls.io/github/ship-components/ship-components-grid)
+[![Coverage Status](https://coveralls.io/repos/github/ship-components/ship-components-grid/badge.svg)](https://coveralls.io/github/ship-components/ship-components-grid)
```

Comment:
I use shields.io because of my ocd. It ensures all the badges are the same. There should be a shields.io link for coveralls